src/main/java/pl/wedding/management/model/employee/EmployeeDto.java

```diff
@@ -0,0 +1,18 @@
+package pl.wedding.management.model.employee;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import java.math.BigDecimal;
+
+@Builder
+@Getter
+class EmployeeDto {
```

Comment:
Wszystkie DTO w architekturze czy to modularnej czy to hexagonalnej muszą być publiczne.

Reply:
to będzie do poprawki, przy tym commicie wywaliło mi public